lib/streams/recordStream.js

```diff
@@ -65,14 +59,52 @@ function createRecordStream( filePath, dirPath ){
     done();
   };
 
-  return fs.createReadStream( filePath )
-    .pipe( csvParser )
+  return fileStreamDispatcher(fs.createReadStream( filePath ), filePath)
     .pipe( contentHashStream )
     .pipe( validRecordFilterStream )
     .pipe( cleanupStream )
     .pipe( documentStream );
 }
 
+function geojsonStream(stream) {
+  return stream
+    .pipe(split())
+    .pipe(through.obj((line, _enc, next) => {
+      let row;
+      try {
+        const geojson = JSON.parse(line);
+        row = {
+          NUMBER: _.get(geojson, 'properties.number'),
+          STREET: _.get(geojson, 'properties.street'),
+          LON: _.get(geojson, 'geometry.coordinates[0]'),
```

Comment:
Would it be worth adding a quick check that the geometry type is `Point` here?

I'm not really sure what the reasons were for moving to `.geojson` but presumably they are so they can expand this format to do things like centerlines which would be `LineString` and potentially cause issues here? 🤷‍♂️

Reply:
<img width="653" alt="Screenshot 2021-02-24 at 15 14 15" src="https://user-images.githubusercontent.com/738069/108936150-3415ce80-76b3-11eb-9afc-8ce6d52d51b8.png">

🔮